optimum/intel/openvino/quantization.py

```diff
@@ -234,17 +235,26 @@ def __init__(self, request):
                 self.request = request
 
             def __call__(self, *args, **kwargs):
-                data_cache.append(*args)
+                data_cache.append(deepcopy(*args))
                 return self.request(*args, *kwargs)
 
+            def start_async(
+                self,
+                inputs: Any = None,
+                userdata: Any = None,
+                shared_memory: bool = False,
+            ) -> None:
+                data_cache.append(inputs)
+                self.request.start_async(inputs, userdata, shared_memory)
+
             def __getattr__(self, attr):
                 if attr in self.__dict__:
                     return getattr(self, attr)
                 return getattr(self.request, attr)
 
         self.model.request = InferRequestWrapper(self.model.request)
-        for i, data in enumerate(calibration_dataloader):
-            self.model.generate(**data, max_new_tokens=10)
+        for _, data in enumerate(calibration_dataloader):
+            self.model.generate(**data, max_new_tokens=100)
```

Comment:
Is this modification added to reduce accuracy degradation resulting from quantization? If yes, what did you observe when varying this parameter ?

Reply:
In the process, and will update a bit later.